packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -133,9 +131,7 @@ class Dart2JSTarget extends Target {
       ? PackageMap.globalGeneratedPackagesPath
       : PackageMap.globalPackagesPath;
     final File outputFile = environment.buildDir.childFile('main.dart.js');
-    if (Platform.environment.containsKey('FLUTTER_WEB_USE_SKIA')) {
-      environment.defines['FLUTTER_WEB_USE_SKIA'] = Platform.environment['FLUTTER_WEB_USE_SKIA'];
-    }
+    final bool useSkia = environment.defines['FLUTTER_WEB_USE_SKIA'] == 'true';
```

Comment:
nit: create a constant with a description of what it does. Also follow the existing naming convention: `FlutterWebUseSkia` or more accurately (because everything is flutter here) `WebUseSkia`

Reply:
Thank you!